tests/logs/RegressionTests_ursa.log

```diff
@@ -1,15 +1,23 @@
 ====START OF URSA REGRESSION TESTING LOG====
 
 UFSWM hash used in testing:
+<<<<<<< HEAD
```

Comment:
Please resolve the conflicts in the log.

Reply:
I've recently chatted with Kai, he's rerunning on Ursa, so it's OK to ignore that for now and he'll upload the updated one.

Reply:
@DeniseWorthen Rerunning the RT now and will update the log ASAP.